src/NLog/LayoutRenderers/StackTraceFormat.cs

```diff
@@ -52,5 +52,10 @@ public enum StackTraceFormat
         /// Detailed flat format (method signatures displayed in a single line).
         /// </summary>
         DetailedFlat,
+
+        /// <summary>
+        /// Raw format (multiline with source info - as returned by StackFrame.ToString() method).
+        /// </summary>
+        RawWithSource,
```

Comment:
I'm not using this LayoutRenderer, so fixing the existing Raw-format is fine with me.